src/itree.cpp

```diff
@@ -122,6 +122,19 @@ int main(int argc, char** argv){
         auto path = correct_path(vm, env);
         irods::experimental::client_connection conn;
 
+        const auto s = fs::client::status(conn, path);
+
+        if (!fs::client::exists(s))
+        {
+            std::cerr << fmt::format("Error: logical path [{}] does not exist.",path.c_str()) << std::endl;
+            return 1;
+        }
```

Comment:
This isn't necessary given the more specific existence check below.

Reply:
Yep, done.